pipelines/upload_transacao_cct/utils.py

```
@@ -225,3 +251,103 @@ def merge_final_data(cur: cursor, blob: Blob, full_refresh: bool):
+        MERGE INTO public.{constants.LOG_TABLE_NAME.value} AS t
+        USING (
+            VALUES
+            {','.join(date_values)}
+        ) AS s(data)
+        ON t.data = s.data
+        WHEN MATCHED THEN
+            UPDATE SET datetime_alteracao = now()
+        WHEN NOT MATCHED THEN
+            INSERT (data, datetime_alteracao)
+            VALUES (s.data, now());
+    """
```

Ruff (0.14.1)

Possible SQL injection vector through string-based query construction

(S608)